packages/dbgpt-serve/src/dbgpt_serve/auth/tests/test_management_service.py

```
@@ -0,0 +1,431 @@
+    service.login("alice", "new-password", "", "")
+
+
+def test_account_set_deactivation_requires_current_impact(service, admin):
+    account_set = create_account_set(service, admin)
+    create_query_user(service, admin, account_set.account_set_id)
+    with db.session() as session:
+        for table_name in ("connect_config", "knowledge_space", "gpts_app"):
+            session.execute(
+                text(
+                    f"INSERT INTO {table_name} (account_set_id) "
+                    "VALUES (:account_set_id)"
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)